modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -642,7 +642,7 @@ CV_EXPORTS Mat findHomography( InputArray srcPoints, InputArray dstPoints,
 @param Qz Optional output 3x3 rotation matrix around z-axis.
 
 The function computes a RQ decomposition using the given rotations. This function is used in
-decomposeProjectionMatrix to decompose the left 3x3 submatrix of a projection matrix into a camera
```

Comment:
During the merge of this PR we need to define policy how to provide documentation links.
And avoid mess between `#` and `@ref` (and related merge conflicts).

4.x branch has links already through `#`:
- [page](https://docs.opencv.org/4.6.0/d9/d0c/group__calib3d.html#ga044b03d81f99370db05c52c3d0b46599)
- [source](https://github.com/opencv/opencv/blame/4.6.0/modules/calib3d/include/opencv2/calib3d.hpp#L757)